production/helm/loki/values.yaml

```diff
@@ -629,6 +657,15 @@ write:
   extraEnvFrom: []
   # -- Lifecycle for the write container
   lifecycle: {}
+    # -- The /flush_shutdown preStop hook is recommended as part of the ingester
```

Comment:
instead of just a comment, how would you feel about taking an opinionated stance here? 

I would like as a default to set the `lifecycle` on line 107 of the write template to have the `preStop` hook, but with an option to override via `values.yaml`. For example, maybe we make the default value here `null`, and have a template that, if `null` and autoscaling, we include the default `preStop`, but if not `null` and autoscaling, we include whatever was provided in the `values.yaml`.

similarly, if not `null` and not autoscaling, we can include whatever was provided, but if `null` and not autoscaling we don't include anything?

Reply:
sure, updated